Build.md

```diff
@@ -0,0 +1,123 @@
+# Build
+
+The following paragraphs describe step by step how to build `neovim-qt`.
+
+## Ubuntu 18.04
+
+Get the `neovim-qt` source code either via git
+
+    $ sudo apt install -y git`
+    $ git clone https://github.com/equalsraf/neovim-qt.git
+    $ cd neovim-qt
+
+or download the zip file.
+
+    $ sudo apt install -y wget unzip
+    $ wget -c https://github.com/equalsraf/neovim-qt/archive/master.zip
+    $ unzip master.zip
+    $ cd neovim-qt-master
+
+Navigate to the `neovim-qt` source code directory and install the necessary dependencies (Qt, `cmake, compiler and `neovim`).
```

Comment:
Backtick typo on this line?

```
`Qt`, `cmake`, `compiler` and `neovim`
```

Reply:
I tried to use the backticks for program names only.

Reply:
Sure, but I think there is still a typo :)

```
(Qt, `cmake`, compiler and `neovim`)
```